Classes/Issues/Branches/IssueTargetBranchModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  IssueTargetBranchModel.swift
+//  Freetime
+//
+//  Created by Yury Bogdanov on 13/03/2018.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+final class IssueTargetBranchModel: ListDiffable {
+    
+    let targetBranchText: NSAttributedStringSizing
+
+    
+    init(branch: String, width: CGFloat) {
+        
+        let branchTitleAttributes: [NSAttributedStringKey: Any] = [
+            .font: Styles.Text.secondaryCode.preferredFont,
+        ]
+        let titleAttributes: [NSAttributedStringKey: Any] = [
+            .font: Styles.Text.secondaryBold.preferredFont
```

Comment:
Could we change this to just `secondary`? I don't think the title should be bold.

Reply:
Sure! Changed it `secondary`, looks a little bit better :)